Source/Eto.WinForms/WinConversions.cs

```diff
@@ -403,7 +403,7 @@ public static MouseButtons ToEto(this swf.MouseButtons button)
 
 		public static Graphics ToEto(this sd.Graphics g)
 		{
-			return new Graphics(new GraphicsHandler(g));
+			return new Graphics(new GraphicsHandler(g, false));
```

Comment:
When called from DrawableHandler.OnPaint, this will dispose the graphics object which is incorrect.